modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="olm-enabling-operator-for-restricted-network_{context}"]
+= Enabling your Operator for restricted network environments
+
+As an Operator author, your CSV must meet additional requirements for your
```

Comment:
Not critical, and it might be too much of a mouthful, but I'd prefer reading something like "the following additional requirements" since you're following it up with the list.

Reply:
No you're right, I think it had it in an earlier iteration. Adding back and trimmed up the rest a bit.